requirements/dev.txt

```diff
@@ -22,13 +22,24 @@ click==8.1.0
     # via
     #   -r test.txt
     #   black
+    #   mkdocs
+    #   mkdocs-click
     #   pip-tools
+<<<<<<< HEAD
 docker==6.1.3
     # via -r test.txt
 idna==3.4
+=======
```

Comment:
There seem to be conflicts in the requirements that have not been resolved. Please update this as well.

Reply:
Done.